sirepo/package_data/static/js/raydata.js

```diff
@@ -672,6 +750,17 @@ SIREPO.app.directive('scanSelector', function() {
 
 	    $scope.getScanField = raydataService.getScanField;
 
+	    $scope.hoverChange = (index, hovered) => {
+		if (! hovered) {
+		hoveredIndex = null;
+		return;
+		}
+		hoveredIndex = index;
+	    };
+
+	    $scope.newCommand = function() {
+		$('#' + panelState.modalId('columnPicker')).modal('show');
```

Comment:
Create a function in `raydataService` for getting the modal object.

Reply:
e64969d8a3a9303ebcf97566971db0411515341f